ingest/usecase/plugins/orderbook/claimbot/config.go

```diff
@@ -0,0 +1,62 @@
+package claimbot
+
+import (
+	"github.com/osmosis-labs/sqs/delivery/grpc"
+	authtypes "github.com/osmosis-labs/sqs/domain/cosmos/auth/types"
+	sqstx "github.com/osmosis-labs/sqs/domain/cosmos/tx"
+	"github.com/osmosis-labs/sqs/domain/keyring"
+	"github.com/osmosis-labs/sqs/domain/mvc"
+	orderbookdomain "github.com/osmosis-labs/sqs/domain/orderbook"
+	orderbookgrpcclientdomain "github.com/osmosis-labs/sqs/domain/orderbook/grpcclient"
+	"github.com/osmosis-labs/sqs/log"
+
+	txfeestypes "github.com/osmosis-labs/osmosis/v26/x/txfees/types"
+
+	txtypes "github.com/cosmos/cosmos-sdk/types/tx"
+)
+
+// Config is the configuration for the claimbot plugin
+type Config struct {
+	Keyring             keyring.Keyring
+	PoolsUseCase        mvc.PoolsUsecase
+	OrderbookUsecase    mvc.OrderBookUsecase
+	OrderbookRepository orderbookdomain.OrderBookRepository
```

Comment:
We should really only need the use case. Usecase encapsulates the repository. So all requests to the repository should be proxied through the use case.

This is an implementation of the MVC design pattern where the repository is the model.

Reply:
Thanks, addressed as per your suggestion!